src/core/style/_palette.scss

```diff
@@ -335,7 +335,9 @@ $md-light-theme-background: (
   hover:             rgba(black, 0.04), // TODO(kara): check style with Material Design UX
   card:       white,
   dialog:     white,
-  disabled-button:   rgba(black, 0.12)
+  disabled-button:   rgba(black, 0.12),
+  slide-toggle-bar: map_get($md-grey, 500),
+  slide-toggle-thumb: map_get($md-grey, 400)
```

Comment:
Yes. I had that before. Checking for dark, is not needed, since the slide-toggle always uses the same hues for dark or light themes. Added a comment for :)